nilearn/_utils/glm.py

```diff
@@ -145,6 +145,10 @@ def z_score(pvalue):
     """ Return the z-score corresponding to a given p-value.
     """
     pvalue = np.minimum(np.maximum(pvalue, 1.e-300), 1. - 1.e-16)
+    # if np.any(pvalue > .5):
```

Comment:
why is the change commented out?

Reply:
Because, for the moment, my goal is to reproduce the error in a test function. Then, I'll try to fix the problem itself. These lines were just a first attempt to solve the problem.